util/langfuse.py

```
@@ -0,0 +1,56 @@
+    if _initialized:
+        return
+
+    try:
+        environment = os.environ.get("ENVIRONMENT_NAME")
+        if environment and not os.environ.get("LANGFUSE_SECRET_KEY"):
+            ssm_parameter = f"{environment}-langfuse-secret-key"
+            secret_key = get_parameter(ssm_parameter)
+            if secret_key:
+                os.environ["LANGFUSE_SECRET_KEY"] = secret_key
+        _initialized = True
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)